DependencyInjection/DoctrineExtension.php

```diff
@@ -72,6 +72,20 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->adapter->loadServicesConfiguration($container);
 
+        // Use a prototype when the ORM is available.
+        // This avoids listing all ORM commands by hand.
+        if (class_exists(Version::class) && method_exists(XmlFileLoader::class, 'registerClasses')) {
```

Comment:
` && method_exists(XmlFileLoader::class, 'registerClasses')` part of the condition could be removed now that the DI dependency has been bumped to 3.3.